.eslintrc.json

```diff
@@ -1,7 +1,56 @@
 {
-  "extends": ["next", "next/core-web-vitals", "plugin:prettier/recommended", "plugin:jsx-a11y/recommended"],
-  "plugins": ["prettier", "jsx-a11y"],
+  "extends": [
+    "airbnb",
+    "eslint:recommended",
+    "next",
+    "next/core-web-vitals",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:jest/recommended",
+    "plugin:jsx-a11y/recommended",
+    "plugin:prettier/recommended",
+    "plugin:react-hooks/recommended",
+    "plugin:react/recommended",
+    "plugin:security/recommended-legacy",
+    "plugin:sonarjs/recommended-legacy",
+    "plugin:unicorn/recommended"
+  ],
+  "plugins": [
+    "@typescript-eslint",
+    "html",
+    "import",
+    "jest",
+    "jsx-a11y",
+    "prettier",
+    "react",
+    "react-hooks",
+    "security",
+    "sonarjs",
+    "unicorn"
+  ],
   "rules": {
-    "prettier/prettier": "error"
+    "no-restricted-syntax": "off",
```

Comment:
This is off because it does not like iterator functions. Which we are using in the unit tests to mock `GetFormData`from next.